tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py

```diff
@@ -11,145 +11,376 @@
     preparsing_validators=[
         validators.hasLength(156),
     ],
-    postparsing_validators=[],
+    postparsing_validators=[
+            validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                  result_field='RACE_HISPANIC', result_function=validators.oneOf((1, 2)),
+            ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='RACE_AMER_INDIAN', result_function=validators.oneOf((1, 2)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='RACE_ASIAN', result_function=validators.oneOf((1, 2)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='RACE_BLACK', result_function=validators.oneOf((1, 2)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='RACE_HAWAIIAN', result_function=validators.oneOf((1, 2)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='RACE_WHITE', result_function=validators.oneOf((1, 2)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='MARITAL_STATUS', result_function=validators.oneOf((1, 2, 3, 4, 5)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                      result_field='PARENT_WITH_MINOR_CHILD', result_function=validators.oneOf((1, 2, 3)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='EDUCATION_LEVEL', result_function=validators.oneOf((1, 2, 3, 4, 5,
```

Comment:
so, some of these are interesting. `EDUCATION_LEVEL` for instance has the following field validators:
```
validators.or_validators(
    validators.isInLimits(0, 16),
    validators.isInLimits(98, 99)
)
```
then the preparsing validators basically says if `FAMILY_AFFILIATION` is `1`, `2`, or `3`, then `EDUCATION_LEVEL` can't be `0`. could the preparsing validator be simplified?

Reply:
Right, in fact we can simplify this for cross validation to "not equal to zero" since all other cases are covered by the field validation